### PR TITLE
Updated extensions-chromium/publish/publish-extension

### PR DESCRIPTION
**Pull request by jm-trd-ms**

### CODE REVIEW

File: microsoft-edge/extensions-chromium/publish/publish-extension.md [hunk 1]
```diff
@@ -70,16 +69,16 @@ Define the specific markets in which you plan to offer your extension.  The defa
...
-| Website URL | A web page that provides additional information about your extension.  The `Website URL` must point to a page on your own website, not the web listing for your extension in the Microsoft Edge Add-ons store.  The `Website URL` helps users learn more about your extension, its features, and any other relevant information.  |  
+| Website URL | A web webpage that provides additional information about your extension.  The `Website URL` must point to a webpage on your own website, not the web listing for your extension in the Microsoft Edge Add-ons store.  The `Website URL` helps users learn more about your extension, its features, and any other relevant information.  |
```
Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/publish/publish-extension.md:81]: Change "A web webpage" to "A webpage"

File: microsoft-edge/extensions-chromium/publish/publish-extension.md [hunk 2]
```diff
@@ -106,11 +105,11 @@ The information provided in the following section is displayed to users who revi
...
-| YouTube video URL \(optional\) | You may include a promotional YouTube video of your extension.  The `YouTube video URL` video is displayed on the store listing page of your extension.  |  
+| YouTube video URL \(optional\) | You may include a promotional YouTube video of your extension.  The `YouTube video URL` video is displayed on the store listing webpage of your extension.  |  
 | Short description \(required\) | To edit the `short description`, you must update the description field in your manifest file of your extension package and reupload it.  |  
-| Search terms \(optional\) | `Search terms` are single words or phrases that help users discover your extension when searching in the Microsoft Edge Add-ons store.  The search terms aren't displayed to users.  |  
+| Search terms \(optional\) | `Search terms` are single words or phrases that help discover your extension a user searches in the Microsoft Edge Add-ons store.  The search terms aren't displayed to users.  |
```
Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/publish/publish-extension.md:110]: Change:
single words or phrases that help discover your extension a user searches in the Microsoft Edge Add-ons store
to:
single words or phrases that help discover your extension when a user searches in the Microsoft Edge Add-ons store
or revert to the original wording.

File: microsoft-edge/extensions-chromium/publish/publish-extension.md (hunk 1 above)
Comment (jm-trd-ms) [reply microsoft-edge/extensions-chromium/publish/publish-extension.md:81]: Updated.

File: microsoft-edge/extensions-chromium/publish/publish-extension.md (hunk 2 above)
Comment (jm-trd-ms) [reply microsoft-edge/extensions-chromium/publish/publish-extension.md:110]: Updated.

**Comment by jm-trd-ms** [comment]
#sign-off and merged.

**Pull request merged**